### DISCUSSION TITLE
I would like to creat a chinese phone app (ios, and android) for sports mathing. how much should I pay

### DISCUSSION THREAD

**Discussion by ppppiiinx** [community/community#140602] [Actions]
### Select Topic Area

Question

### Body

I would like to creat a chinese phone app (ios, and android) for sports mathing. how much should I pay

**Comment by juacardonahe** [comment]
The cost of developing a sports matching app for iOS and Android can range from $10,000 to $150,000, depending on the complexity. A basic app with user profiles, matching, and messaging will be on the lower end, while more advanced features like scheduling, real-time tracking, or AI matching can push the price higher. Don’t forget to factor in ongoing costs for maintenance and hosting, which could add $5,000 to $10,000 annually.

**Comment by ghostinhershell** [comment]
Hi @ppppiiinx,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this subject. It's possible that another GitHub user might be able to help but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
